evap/evaluation/models.py

```diff
@@ -512,19 +520,35 @@ class TextAnswer(Answer):
     """A free-form text answer to a question (usually a comment about a course
     or a contributor)."""
 
-    elements_per_page = 5
-
     reviewed_answer = models.TextField(verbose_name=_("reviewed answer"), blank=True, null=True)
     original_answer = models.TextField(verbose_name=_("original answer"), blank=True)
-
-    checked = models.BooleanField(verbose_name=_("answer checked"), default=False)
-    hidden = models.BooleanField(verbose_name=_("hide answer"), default=False)
+    
+    TEXT_ANSWER_STATES = (
+        ('N', _('not published')),
```

Comment:
longer names are no problem, the complete name is never used so far. so also translation are irrelevant.
there is no documentation yet, but [the wiki](https://github.com/fsr-itse/EvaP/wiki/Publishing-Results) will hold this information.
#522 describes why the feature is needed.